### PR TITLE
support `@supports`

### PR DESCRIPTION
**Pull request by jeongoon**
Hi, I'm trying to add @supports.
I made some tests by referring `@media`.

but I don't know how to apply to my `current` setup right away to see the effects as well.
because I'm not a professional programmer.
is there any way to apply in my `stack.yaml` or equivalent one to compile with patched version?

### CODE REVIEW

**Comment by jeongoon** [comment]
I found how to compile yesod with patched shakespeare  [from stackoverflow](https://stackoverflow.com/questions/32849269/how-to-install-use-a-local-version-of-package-using-stack).
compiling now. hopefully work as I wanted.

**Comment by snoyberg** [comment]
Have you had a chance to test this and see if it does what you're looking for?

**Comment by jeongoon** [comment]
Oh, I'm using my patched version now (on archlinux with lts 16.31 which included in scaffolding) 
But i can see some test is failing here.
Any guide line to help?

File: test/Text/CssSpec.hs [hunk 1]
```diff
@@ -278,8 +278,8 @@ bin {
     it "lucius media" $ do
       celper "@media only screen{foo bar{baz:bin}}" $(luciusFile "test/cassiuses/external-media.lucius")
       celper "@media only screen {\n    foo bar {\n        baz: bin;\n    }\n}\n" $(luciusFileDebug "test/cassiuses/external-media.lucius")
-      celper "@media only screen{foo bar{baz:bin}}" [lucius|
-        @media only screen{
+      celper "@media only screen {foo bar{baz:bin}}" [lucius|
```
Comment (snoyberg) [inline test/Text/CssSpec.hs:281]: I'd prefer not to see unrelated changes in this PR. This line specifically is present to test that things work both with and without extra spacing.

**Review by snoyberg — CHANGES_REQUESTED** [review]

**Comment by jeongoon** [comment]
noted. I revert my changes which is note related to "@supports".
thank you.

**Pull request merged**